api_spec.yaml

```diff
@@ -5267,12 +5367,82 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
+  /recipe/multitenancy/connectionuridomain/list/v2:
```

Comment:
also mention the possible factors that can be in the firstFactors and requiredSecondaryFactors